ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1653,8 +1653,18 @@ The configured NIC will be used to access the compute node.\n\n"
 
 	valid=0
 	while [ $valid -ne 1 ]; do
+		{% if usb_type == "hn" %}
+		promptnet "(admin) IP address" "$admin_ip" "admin_ip"
+		admin_ip="$val"
+		# we disallow DHCP on HN install
+		if [ "${admin_ip}" == "dhcp" ]; then
+			echo "The first node cannot have a dynamic admin address during install."
```

Comment:
We should clear the `$admin_ip` variable in this case (`admin_ip=""`).
Sentence update:
- *node* -> *compute node*
- *address* -> *IP address*